trashies.go

```diff
@@ -57,24 +57,45 @@ func DeletePerson(w http.ResponseWriter, r *http.Request) {
 }
 
 func launch(w http.ResponseWriter, r *http.Request) {
-	t, _ := template.ParseFiles("index.html", "footer.html", "header.html")
-	t.ExecuteTemplate(w,"index", nil)
+	t, _ := template.ParseFiles("index.html", "templates/footer.html", "templates/navbar.html", "templates/header.html", "templates/jsimports.html")
+	t.ExecuteTemplate(w, "index", nil)
+}
+
+func join(w http.ResponseWriter, r *http.Request) {
+	t, _ := template.ParseFiles("join.html", "templates/footer.html", "templates/navbar.html", "templates/header.html", "templates/jsimports.html")
```

Comment:
Not sure how this should be done, but should we pull out the parseFiles/executeTemplate into a more general function? Is that possible?

Reply:
it probably is, I wasn't clear on how to do it. Slices are hard.